client/src/components/StepWizard/WizardCheckboxItem.js

```diff
@@ -73,7 +75,7 @@ export const WizardCheckboxItem = ({ text, checked, onChange, ...props }) => {
       {...props}
     >
       <StyledCheckbox checked={checked} />
-      <span className="text">{text}</span>
+      <StyledSpan className="text">{text}</StyledSpan>
```

Comment:
We can probably remove the `className="text"` here, I don't think it is being used anywhere.